README.md

```diff
@@ -1 +1,26 @@
 Client-server implementation of UCLA LCD lab event segmentation study.
+
+## Usage
+
+### Install some global packages
+
+`npm install -g live-server pm2`
+
+### Setup Server
+
+Run the following command under server folder
+- Install packages: `npm install`
+- Start server in the background: `pm2 start app.js --name "event-segmentation-server"`
+
+### Setup Client
+
+Run the following command under client folder
+- Install packages: `npm install` 
+- Start the client in the background as well: `pm2 start live-server --name "event-segmentation-client"`
```

Comment:
This is just my speculation of how the client is supposed to be used. If I'm wrong, feel free to adjust the command.